apiserver/stateauthenticator/auth.go

```diff
@@ -154,16 +145,23 @@ func (a *Authenticator) AuthenticateLoginRequest(
 	if err == nil {
 		return authInfo, err
 	}
+
+	sysState, err := a.statePool.SystemState()
+	if err != nil {
+		return authentication.AuthInfo{}, errors.Trace(err)
+	}
+	authInfo.PermissionsFn = userPermissions(sysState)
```

Comment:
Not sure is this is right, will need to check logic a bit more closely. ie we don't want this for the jwt case

Reply:
We talked about this and were happy with what is going on.